src/main.zig

```diff
@@ -917,6 +920,11 @@ fn buildOutputType(
                         install_name = args_iter.next() orelse {
                             fatal("expected parameter after {s}", .{arg});
                         };
+                    } else if (mem.startsWith(u8, arg, "--compress-debug-sections=")) {
```

Comment:
Just noticed it now: is there any reason for having `=` in the option? Would `--compress-debug-section [none|zlib]` not suffice?

Reply:
I was aiming for compatibility with the linker flag:

```
$ man ld.lld-13| grep -E '^ +--compress-debug-sections'
     --compress-debug-sections=value
motiejus@mtmine:~$ man ld| grep -E '^ +--compress-debug-sections'
       --compress-debug-sections=none
       --compress-debug-sections=zlib
       --compress-debug-sections=zlib-gnu
       --compress-debug-sections=zlib-gabi
           --compress-debug-sections=none doesn't compress <...>
```